soccer/gameplay/tactics/defense.py

```diff
@@ -61,7 +76,21 @@ def debug(self):
     def debug(self, value):
         self._debug = value
 
-    def execute_running(self):
+    def execute_defending(self):
+
+        defender1 = self.subbehavior_with_name('defender1')
+        defender2 = self.subbehavior_with_name('defender2')
+        close_defender = defender1
+        if (main.ball().pos.mag() < constants.Field.ArcRadius * 2):
+            if (defender1 != None and defender2 != None):
```

Comment:
The Defense behavior is setup so that it always creates two SubmissiveDefender behaviors.  They're marked as optional, so there are definitely times when a robot is not assigned to the behavior, but the behavior is still always there.